client/src/components/AlumniWorkspace.js

```diff
@@ -11,13 +16,68 @@ export default class AlumniWorkspace extends Component {
         super(props)
         this.state = {
             activeItem: 'collegesAccepted',
-            collegesAccepted: []
+            collegesAccepted: [],
+            
         }
     }
 
     handleMenuClick = (e, { id }) => this.setState({ activeItem: id })
 
-    render() {
+    componentWillMount(){
+        this.getCollegesAcceptedInto();
+    }
+
+    getCollegesAcceptedInto = () => {
+        const theEndPoint = `/alumni/collegesAcceptedInto/all/${this.props.userDetails._id}`;
+        makeCall({}, theEndPoint, `GET`)
+        .then(res => {
+            this.setState({
+                collegesAccepted: res
+            })
+        })
+        .catch(e => {
+            console.log('Error #getCollegeAcceptedInto', e)
```

Comment:
nit: console.error

tangential question should we handle error logging like this on the back end? it seems like this wouldn't especially be helpful user info.